### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by ozimeu**
Cleaning up here and there

### CODE REVIEW

File: controllers/controller.rb [hunk 1]
```diff
@@ -44,21 +31,60 @@ class Controller < Sinatra::Base
...
+    @notifier = SlackNotifier.new(slack_token: AppConfig.slack.client_token,
+                                  channel: AppConfig.slack.channel,
+                                  twilio_name: AppConfig.twilio.client_name)
+  end
+
+  def instantiate_twilio_client
+    @twilio_client = SetupTwilioClient.new(account_sid: AppConfig.twilio.account_sid,
+                                           auth_token: AppConfig.twilio.auth_token)
+                                      .call
+  end
```
Comment (inclooder) [inline controllers/controller.rb]: Another approach is to lazy load @notifier and @twilo_client like that
```
def notifier
  @notifier ||= SlackNotifier.new(...)
end

def twilio_client
  @twilo_client ||= SetupTwilioClient.new(...)
end
```

After that you could remove `attr_reader and initialize method`

File: services/check_line_busy.rb [hunk 2]
```diff
@@ -8,14 +8,37 @@ def initialize(client:)
...
-    in_progress_conference = conferences.list(status: 'in-progress').first
-    in_progress_conference.participants.list.count == 2 if in_progress_conference
+  def line_busy?
+    return false unless find_in_progress_conference
+    check_current_conference_participants
+  end
+
+  def check_current_conference_participants
+    participants = conference_participants
+    participants.list.count == 2
```
Comment (inclooder) [inline services/check_line_busy.rb]: You could just say `conference_participants.list.count == 2` ?

File: services/check_line_busy.rb [hunk 3]
```diff
@@ -8,14 +8,37 @@ def initialize(client:)
...
+  end
+
+  def find_in_progress_conference
+    conferences = fetch_client_conferences
+    conferences.list(status: 'in-progress').first
+  end
+
+  def conference_participants
+    in_progress_conference = find_in_progress_conference
+    in_progress_conference.participants
```
Comment (inclooder) [inline services/check_line_busy.rb]: Same thing

File: services/check_line_busy.rb [hunk 4]
```diff
@@ -8,14 +8,37 @@ def initialize(client:)
...
+  end
+
+  def fetch_client_conferences
+    account = client_account
+    account.conferences
```
Comment (inclooder) [inline services/check_line_busy.rb]: same as up

Comment (ozimeu) [inline services/check_line_busy.rb]: Yeah, it looks like it's too much, I'll fix these 3 similar issues in a minute ;-)

File: controllers/controller.rb (hunk 1 above)
Comment (ozimeu) [inline controllers/controller.rb]: Yes, the approach you've suggested is also a good idea. I just wanted to explicitly point out here that we're instantiating these two objects as the main parts of this application

**Pull request merged**